sdk/core/azure-core-tracing-opentelemetry/src/samples/ListKeyVaultSecretsTracingSample.md

```diff
@@ -9,12 +9,12 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 <dependency>
     <groupId>io.opentelemetry</groupId>
     <artifactId>opentelemetry-sdk</artifactId>
-    <version>0.2.4</version>
+    <version>0.6.0</version>
```

Comment:
Not sure why we have so many .md files in /src/samples that have the same getting started section. Should this be consolidated into a single README.md file in /src/samples and then have Java code samples referenced in README? It would be consistent with what we do in other client libraries.

Reply:
Yeah, it was a solution to make this library lightweight and not have those client lib dependencies. We kept running into dependency issues for this library having to wait till all other "dependent" libraries to be out first.

Reply:
Adding these dependencies in `test` scope should not increase the size of the library. Samples only need test scope dependencies.

Reply:
Adding client library dependencies in test scope would still require this package to be released only after all other packages are released.
We could possibly consider using older versions of client dependencies to not run into the waiting cycle for releasing this package.
Would give it a try updating the samples in https://github.com/Azure/azure-sdk-for-java/issues/13423